manifest.json

```diff
@@ -1,15 +1,14 @@
 {
-  "name": "Daemon Disqualifier",
-  "description": "Watches user activity on issues, sends reminders on deadlines, and unassign inactive users.",
-  "ubiquity:listeners": ["pull_request_review_comment.created", "issue_comment.created", "push"],
+  "name": "User activity watcher",
```

Comment:
I thought it's supposed to match the repository name for the plugin installer or something.

Reply:
Good catch, my mistake when merging the JSON from the main branch.